developer_setup/debugging.md

```diff
@@ -0,0 +1,64 @@
+## Debugging ManageIQ with Remote Pry
+
+Interactive debugging using pry or byebug is normally difficult with ManageIQ, as `rake evm:start` launches the server and worker processes in the background. Therefore the only way to do interactive debugging is with `pry-remote` or `byebug` running in remote mode.
+
+Running ManageIQ with `byebug` was exceedingly slow, so this guide will focus on remote debugging using `pry`.
+
+### Adding dependencies
+
+You'll need to add the following gems to your `Gemfile`:
+
+```ruby
+  group :development, :test do
+    # ...
+    gem 'pry'
+    gem 'pry-remote'
+    gem 'pry-nav'
+    # ...
+  end
+```
+
+The `pry-remote` gem must be required at some point before your (first) breakpoint. A convenient place to put this is in `config/environments/development.rb`:
+
+```ruby
+require 'pry-remote'
+```
```

Comment:
Is this part necessary?  When you do `gem 'pry-remote'`, then require 'pry-remote' is done automatically, unless you do `:require => false`.

Reply:
for some reason, when i tried without `require`ing, i got `undefined local variable` errors. maybe I was doing something wrong?

Reply:
Yeah something is fishy here. You shouldn't need to re-require this. Double check?

Reply:
tried it again and no longer had this issue. not sure how i encountered it the first time. anyway, removed from readme :)